### PR TITLE
[WIP] DNS records from DNSAAS

### PR DESCRIPTION
**Pull request by szok**
- [ ] unittests

### CODE REVIEW

File: src/ralph/data_center/admin.py [hunk 1]
```diff
@@ -37,6 +37,7 @@
 from ralph.data_center.models.virtual import Database, VIP
 from ralph.data_center.views.ui import DataCenterAssetSecurityInfo
 from ralph.data_importer import resources
+from ralph.dns.views import DnsView
```
Comment (mkurek) [inline src/ralph/data_center/admin.py]: DNS is abbreviation, so IMO (:sunglasses:) we should use capital letters for it (ex. `DNSView`)

Comment (ar4s) [inline src/ralph/data_center/admin.py]: +1

File: src/ralph/dns/models.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+from ralph.data_center.models.physical import DataCenterAsset
+
+
+class RecordType(Choices):
+    _ = Choices.Choice
+
+    a = _('A')
+    txt = _('TXT')
+    cname = _('CNAME')
+    mx = _('MX')
```
Comment (mkurek) [inline src/ralph/dns/models.py]: we should consider only a, txt and cname IMO

**Comment by mkurek** [comment]
Today is meme (documentation?) day :)



Could you @szok add some briefly documentation about Ralph DNS management? Just to start the document, we'll improve it during later work on DNS feature.

File: src/ralph/dns/forms.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        result = []
+        url = '{}/{}/?{}'.format(
+            settings.DNSAAS_URL,
+            'api/records',
+            urlencode({'type': 'A'})  # TODO add filter for ipaddress
+        )
+        headers = {'Authorization': 'Token {}'.format(settings.DNSAAS_TOKEN)}
+        request = requests.get(url, headers=headers)
+        json_data = request.json()
+        api_results = json_data.get('results', [])
```
Comment (mkurek) [inline src/ralph/dns/forms.py:28]: remember about pagination

Comment (ghost) [inline src/ralph/dns/forms.py:28]: :+1:

File: src/ralph/dns/forms.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        # if json_data['next']:
+        api_results = json_data.get('results', [])
+        if api_results:
+            for item in api_results:
+                result.append(
+                    DNSRecord(
+                        name=item['name'],
+                        type=RecordType.from_name(item['type'].lower()).id,
+                        content=item['content'],
+                        ptr=item['type'] == 'PTR'
```
Comment (szok) [inline src/ralph/dns/forms.py:36]: to delete

**Pull request closed without merging**